certified-connectors/Jupyrest/apiDefinition.swagger.json

```diff
@@ -0,0 +1,294 @@
+                  "x-ms-dynamic-schema": {
+                    "operationId": "GetNotebookMetadata",
+                    "value-path": "input",
+                    "parameters": {
+                      "notebook": {
+                        "parameter": "notebook"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-properties": {
+                    "operationId": "GetNotebookMetadata",
+                    "itemValuePath": "input",
+                    "parameters": {
+                      "notebook": {
+                        "parameterReference": "parameters/notebook"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NotebookResponse"
+            },
+            "headers": {}
+          },
+          "202": {
+            "description": "Success",
+            "headers": {
+              "execution_id": {
+                "type": "string",
+                "description": "execution id"
+              },
+              "location": {
+                "type": "string",
+                "description": "url location for getting execution status"
+              },
+              "retry-after": {
+                "type": "string",
+                "description": "retry after in seconds"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    }
+  },
+  "definitions": {
+    "NotebookResponse": {
+      "title": "NotebookResponse",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "notebook execution id"
+        },
+        "status": {
+          "type": "string",
+          "description": "notebook execution status"
+        },
+        "notebook": {
+          "type": "string",
+          "description": "notebook name"
+        },
+        "parameters": {
+          "type": "object",
+          "description": "notebook input parameters"
+        },
+        "output": {
+          "type": "array",
+          "items": {},
+          "description": "notebook execution output results"
+        },
+        "html": {
+          "type": "string",
+          "description": "notebook html results"
+        },
+        "start_time": {
+          "type": "string",
+          "description": "notebook execution start time"
+        },
+        "end_time": {
+          "type": "string",
+          "description": "notebook execution end time"
+        },
+        "exception": {
+            "type": "string",
+            "description": "notebook execution exception"
+        }
+      }
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "tags": [
+    {
+      "name": "JupyrestConnector",
+      "description": ""
```

Comment:
@litiany23 Empty property values can be removed until unless you have a specific reason to keep them.

Reply:
done